### PR TITLE
Add langfuse integration tile

### PR DESCRIPTION
**Pull request by annthurium**
as per https://github.com/deepset-ai/haystack-core-integrations/issues/709
- add Langfuse integration tile
- Add logo

### CODE REVIEW

File: integrations/langfuse.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
+---
+layout: integration
+name: langfuse
+description: langfuse-haystack integrates tracing capabilities into [Haystack](https://github.com/deepset-ai/haystack) (2.x) pipelines using [Langfuse](https://langfuse.com/). This package enhances the visibility of pipeline runs by capturing comprehensive details of the execution traces, including API calls, context data, prompts, and more. Whether you're monitoring model performance, pinpointing areas for improvement, or creating datasets for fine-tuning and testing from your pipeline executions, langfuse-haystack is the right tool for you.
+
+authors:
+    - name: deepset
+      socials:
+        github: deepset-ai
+        twitter: haystack_ai
```
Comment (annthurium) [inline integrations/langfuse.md:10]: why do all the "socials" point to the deepset twitter rather than Haystack? 🤔

Comment (vblagoje) [reply integrations/langfuse.md:10]: Probably legacy, Tuana recently started the Haystack account

File: integrations/langfuse.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
+---
+layout: integration
+name: langfuse
+description: langfuse-haystack integrates tracing capabilities into [Haystack](https://github.com/deepset-ai/haystack) (2.x) pipelines using [Langfuse](https://langfuse.com/). This package enhances the visibility of pipeline runs by capturing comprehensive details of the execution traces, including API calls, context data, prompts, and more. Whether you're monitoring model performance, pinpointing areas for improvement, or creating datasets for fine-tuning and testing from your pipeline executions, langfuse-haystack is the right tool for you.
```
Comment (bilgeyucel) [inline integrations/langfuse.md:4]: This text will be shown on the integration card and it's a bit too long to fit into the card. Can you make it shorter?
Example 👇 
image

We can use the longer description in the `Overview` section below

File: integrations/langfuse.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+    messages = [
+        ChatMessage.from_system("Always respond in German even if some input data is in other languages."),
+        ChatMessage.from_user("Tell me about {{location}}"),
+    ]
+
+    response = pipe.run(
+        data={"prompt_builder": {"template_variables": {"location": "Berlin"}, "prompt_source": messages}}
+    )
+    print(response["llm"]["replies"][0])
+    print(response["tracer"]["trace_url"])
```
Comment (bilgeyucel) [inline integrations/langfuse.md:155]:
```suggestion
pipe = Pipeline()
pipe.add_component("tracer", LangfuseConnector("Chat example"))
pipe.add_component("prompt_builder", DynamicChatPromptBuilder())
pipe.add_component("llm", OpenAIChatGenerator(model="gpt-3.5-turbo"))

pipe.connect("prompt_builder.prompt", "llm.messages")
messages = [
    ChatMessage.from_system("Always respond in German even if some input data is in other languages."),
    ChatMessage.from_user("Tell me about {{location}}"),
]

response = pipe.run(
    data={"prompt_builder": {"template_variables": {"location": "Berlin"}, "prompt_source": messages}}
)
print(response["llm"]["replies"][0])
print(response["tracer"]["trace_url"])
```

File: integrations/langfuse.md [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+
+authors:
+    - name: deepset
+      socials:
+        github: deepset-ai
+        twitter: haystack_ai
+        linkedin: deepset-ai
+pypi: https://pypi.org/project/langfuse-haystack/
+repo: https://github.com/deepset-ai/haystack-core-integrations/tree/main/integrations/langfuse
+type: Evaluation Framework
```
Comment (bilgeyucel) [inline integrations/langfuse.md:14]:
```suggestion
type: Monitoring Tool
```

**Review by bilgeyucel — COMMENTED** [review]
Thank you @annthurium! I left some minor comments. Additionally, it would be better if you could change the logo and use an image with a background. We are using a dark background and dark texts won't be visible on that background

**Pull request merged**